spec/helpers/suggested_councillors_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
Because we're removing the helper file, we should remove this test file as well. We try not to have pending test files sitting around, they never seem to get filled in.